pool/app/statistics/statistics.mli

```diff
@@ -1,87 +1,179 @@
-module ActiveContacts : sig
-  include Pool_common.Model.IntegerSig
+module System : sig
+  module ActiveContacts : sig
+    include Pool_common.Model.IntegerSig
 
-  val field : Pool_common.Message.Field.t
-end
+    val field : Pool_common.Message.Field.t
```

Comment:
Instead of adding these fields everywhere to the signature, can't we just add it directly to the `IntegerSig`?

`val field : Entity_message.Field.t`